test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -1905,6 +1905,93 @@ void ProtoConverter::buildObjectScopeTree(Object const& _x)
 	m_objectScope.emplace(objectName, node);
 }
 
+string ProtoConverter::optStepToString(Program_OptimizerStep const& _x)
+{
+	switch (_x)
+	{
+	case Program_OptimizerStep_FULLSUITE:
+		return "fullSuite";
```

Comment:
Is it possible to use the name from the class here?

Reply:
`Program_OptimizerStep_FULLSUITE` is an enum that does not have a notion of the actual string name.

Reply:
I know, but at least for the steps that have names as members in their classes, we could use those members